README.md

```diff
@@ -15,51 +15,77 @@ Releases        | <https://github.com/DiamondLightSource/odin-fastcs/releases>
 
 ## Development
 
-1. Clone odin-data and open in its devcontainer (for now, checkout fastcs-dev branch)
-2. Build odin-data into `vscode_prefix`
+In most cases it should be sufficient to work within the odin-fastcs devcontainer (or a
+virtualenv) and run the unit tests to develop new features and bug fixes.
+
+In order to develop and test odin-fastcs alongside local dev versions of odin-control
+and odin-data applications, a development environment can be run up from the odin-data
+devcontainer.
+
+### Full Development Environment
+
+The following instructions assume use of vscode, as it makes it very convenient to set up
+an environment that "just works". However, it is possible to run the same environment up
+manually using the [odin-data-developer] container and the following steps should still
+provide a good overview.
+
+1. Clone [odin-data] (next to odin-fastcs), checkout the `fastcs-dev` branch and re-open
+   in devcontainer
+2. Add odin-fastcs to the workspace using `Workspaces: Add Folder to Workspace...`
```

Comment:
should warn about the error you always get when switching to a workspace inside a container (or is it just me?)